_includes/footer.html

```diff
@@ -5,6 +5,11 @@
 
       <footer>
         <ul class="list-inline">
+          {% if page.path contains 'docs' %}
```

Comment:
Unfortunately, Liquid doesn’t to RegExps. Should I change it to `/docs`, or should I generate the link client-side?